addon/templates/components/cfb-form-editor/question.hbs

```diff
@@ -3,7 +3,7 @@
     {{uk-spinner ratio=2}}
   </div>
 {{else}}
-  {{#with data.lastSuccessful.value.firstObject.node as |model|}}
+  {{#with model as |model|}}
```

Comment:
Habe erst mal das Diff überflogen. Braucht es das `with` hier überhaupt noch?

Ist mehr eine Verständnisfrage.

Reply:
English, please :wink: 

I kept with because there's an "else" branch (if model is null) all the way below. Probably better to use `if/else` explicitly though.

Reply:
Noted :-)

Ok. Sounds reasonable!

Reply:
I'll make the change in #44, since it's based on this PR.